src/test/java/uk/gov/hmcts/opal/controllers/NoteControllerTest.java

```diff
@@ -123,4 +124,24 @@ public void testNotesSearch_NoContent() {
         verify(noteService, times(1)).searchNotes(any(
             NoteSearchDto.class));
     }
+
+    private UserState createUserState() {
```

Comment:
As this is duplicate builder method, can be moved to a helper class and reuse. i.e. 

```
class UserStateBuilder {

public static UserState createUserState() { ... }

}
```

Reply:
Moved to a UserStateBuilder class